### PR TITLE
Enclosure api1600

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Enclosure api1600

### Issues Resolved
N/A

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# v2.6.0 (unreleased)
+### Notes
+- This release supports OneView Rest API versions 800/1000/1200/1600 minimally where we can use OneView v4.10/v4.20/v5.0/v5.2 with this SDK.
```
Comment (chebroluharika) [inline CHANGELOG.md]: make changes in CHANGELOG according to master

Comment (AsisBagga) [reply CHANGELOG.md]: @chebroluharika Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# v2.6.0 (unreleased)
+### Notes
+- This release extends support of the SDK to OneView REST API version 1600 (OneView v5.20).
+
+### Oneview Features supported
+- Encloure
```
Comment (chebroluharika) [inline CHANGELOG.md]: Please correct the spelling to Enclosure and place bugs and enhancements also under 2.6.0

Comment (AsisBagga) [reply CHANGELOG.md]: done!

**Review by chebroluharika — APPROVED** [review]

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**